src/containers/LectureDetailContainers/LectureReviewContainer.js

```diff
@@ -0,0 +1,208 @@
+    display: none; /* Chrome, Safari, Opera*/
+  }
+`;
+const FilterPickSection = styled.div`
+  margin-left: 4px;
+  display: flex;
+  align-items: center;
+  cursor: pointer;
+`;
+const FilterPickLabel = styled.label`
+  margin: 0 2px 0 0;
+  font-size: 12px;
+  font-weight: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  text-align: left;
+  color: ${FontColor};
+`;
+const LowArrowIcon = styled.img.attrs({
+  src:
+    "https://hangang-storage.s3.ap-northeast-2.amazonaws.com/assets/img/LecturesDetailPage/arrow-low.png",
+  alt: "arrow low",
+})`
+  width: 9px;
+  height: 5px;
+  margin-left: 4px;
+`;
+
+const SpinnerWrapper = styled.div`
+  width: 100%;
+  height: 15vh;
+  display: flex;
+  align-items: center;
+`;
+
+/**
+ * @param {*} param0
+ * @returns
+ */
+const LectureReviewContainer = ({ lectureId, lectureReviews, ...rest }) => {
+  const dispatch = useDispatch();
+
+  const { isLoggedIn, isCheckedToken } = useSelector((state) => state.authReducer);
+  const { limit, page, maxPage, sort, isFilterModalOpened, isLoading } = useSelector(
+    (state) => state.lectureDetailReducer
+  );
+  /**
+   * 무한 스크롤
+   * 다음 페이지 리뷰를 호출합니다.
+   */
+  const getMoreReviews = async () => {
+    try {
+      const accessToken = isLoggedIn
+        ? getValueOnLocalStorage("hangangToken").access_token
+        : null;
+
+      const {
+        data: { count, result },
+        status,
+      } = await LectureDetailAPI.getLectureReviews(accessToken, lectureId, {
+        limit: limit,
+        page: page + 1,
+        sort: sort,
+      });
+
+      if (status === 200) {
+        dispatch(addNextPageReviews({ count, result }));
+      }
+    } catch (error) {
+      throw new Error(error);
+    }
+  };
+
+  const fetchMore = debounce((entries) => {
+    const target = entries[0];
+    if (target.isIntersecting && page < maxPage) {
+      getMoreReviews();
+    }
+  }, 300);
+  const { targetRef } = useInfiniteScroll(fetchMore, 5);
+
+  return (
+    <Section>
+      <ReviewInfoSection>
+        <InfoLabel>개인 평가({rest.lectureReviewCount})</InfoLabel>
+        <FilterPickSection onClick={() => dispatch(openFilterModal())}>
+          <FilterPickLabel style={{ cursor: "pointer" }}>{rest.sort}</FilterPickLabel>
+          <LowArrowIcon></LowArrowIcon>
+        </FilterPickSection>
+        {isFilterModalOpened && (
+          <FilterModal
+            lectureId={lectureId}
+            isLoggedIn={isLoggedIn}
+            isCheckedToken={isCheckedToken}
+          />
+        )}
+      </ReviewInfoSection>
+
+      <ReviewWrapper ref={targetRef}>
+        {lectureReviews.result.map((props, idx) => {
+          return <Review key={props.id} idx={idx} props={props} />;
+        })}
+
+        {lectureReviews.count === 0 && (
+          <SubWarningLabel>등록된 개인 평가 정보가 없습니다.</SubWarningLabel>
+        )}
+
+        {isLoading && (
+          <SpinnerWrapper>
+            <LoadingSpinner />
+          </SpinnerWrapper>
+        )}
```

Comment:
조건에 빠르게 만족하는 순서대로 나열하셔야 합니다.

```suggestion
        {isLoading && (
          <SpinnerWrapper>
            <LoadingSpinner />
          </SpinnerWrapper>
        )}

        {!isLoading && lectureReviews.count === 0 && (
          <SubWarningLabel>등록된 개인 평가 정보가 없습니다.</SubWarningLabel>
        )}
                
        {!isLoading && lectureReviews.count !== 0 && lectureReviews.result.map((props, idx) => {
          return <Review key={props.id} idx={idx} props={props} />;
        })}
```